pycarol/apps.py

```diff
@@ -244,3 +263,64 @@ def edit_manifest(self, app_name, manifest):
 
         r = self.carol.call_api(url, method='PUT', data=manifest)
         return r
+
+
+    def get_git_process(self, app_name=None):
+        """
+        Get Git processes definid in the manifest file.
+
+        Args:
+
+            app_name: `str`
+                App name.
+
+        Returns: List of Dicts
+
+        """
+
+        if app_name is None:
+            app_name = self.carol.app_name
+
+        response = self.carol.call_api(path=f'v1/compute/{app_name}/getProcessesByGitRepo',
+                                       method='POST')
+
+        return response
+
+    def build_docker_git(self, git_token, app_name=None, ):
+        """
+        Build App image using manifest definition.
+
+        Args:
+            git_token: `str`
+                Git token to be used to pull the files.
+
+            app_name: `str`
+                App name.
+
+        Returns:
+
+        """
+
+        if app_name is None:
+            app_name = self.carol.app_name
```

Comment:
what about a deprecation warning here?

Reply:
that's ok.. i meant o add a warning or info log entry about the deprecation on this is.. just that

Reply:
I normalize this behavior in all methods and add a comment in the docstrings.